src/ng/compile.js

```diff
@@ -3052,8 +3052,8 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
 
       $compileNode.empty();
 
-      $templateRequest(templateUrl)
-        .then(function(content) {
+      $templateRequest(templateUrl).
```

Comment:
So you're a "dots at the end of the line" kind of guy eh?
This is not in-keeping with the rest of the codebase